integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -400,4 +688,31 @@ private void createAuxiliaryImage(String stageDirPath, String dockerFileLocation
             .command(cmdToExecute))
         .execute(), String.format("Failed to execute", cmdToExecute));
   }
+
+  private void createSecretsForDomain(String adminSecretName, String encryptionSecretName, String domainNamespace) {
```

Comment:
Can we not directly call the method createOcirRepoSecret() and createSecretWithUsernamePassword() directly with out creating a wrapper private method  createSecretsForDomain()

Reply:
We have multiple test methods using the same piece of code. That's why I added a wrapper method.